### PR TITLE
genesis workflow step description

### PR DESCRIPTION
**Pull request by bugzmanov**

### CODE REVIEW

File: internal-api/src/main/scala/com/griddynamics/genesis/util/Describer.scala [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  private def toString(values: Map[String, String]): String = values.map(toString(_)).mkString(", ")
+
+  def describe: String = {
+    if (params.isEmpty) {
+      description
+    } else {
+      "%s (%s)".format(description, toString(params.toMap))
+    }
+  }
+}
```
Comment (ddurnev) [inline internal-api/src/main/scala/com/griddynamics/genesis/util/Describer.scala:62]: Could we use ToStringBuilder from apache commons-lang?

File: jclouds/src/main/scala/com/griddynamics/genesis/jclouds/step/JCloudsStep.scala [hunk 2]
```diff
@@ -43,6 +52,10 @@ case class DestroyVm(roleName: String,
   def isGlobal = true
 
   def roles = Set(roleName)
+
+  override val stepDescription = "Virtual mashine(s) desctruction"
```
Comment (ddurnev) [inline jclouds/src/main/scala/com/griddynamics/genesis/jclouds/step/JCloudsStep.scala]: Typo: shouid read "Virtual machine(s) destruction"

File: internal-api/src/main/scala/com/griddynamics/genesis/util/Describer.scala (hunk 1 above)
Comment (bugzmanov) [inline internal-api/src/main/scala/com/griddynamics/genesis/util/Describer.scala:62]: ToStringBuilder does not support Options and Maps

File: jclouds/src/main/scala/com/griddynamics/genesis/jclouds/step/JCloudsStep.scala (hunk 2 above)
Comment (bugzmanov) [inline jclouds/src/main/scala/com/griddynamics/genesis/jclouds/step/JCloudsStep.scala]: fixed

File: internal-api/src/main/scala/com/griddynamics/genesis/util/Describer.scala [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    params(key) = "[%s]".format(value.mkString(", "))
+    this;
+  }
+
+  def param(key: String, values: Map[String, String]): Describer = {
+    params(key) = "{%s}".format(toString(values))
+    this;
+  }
+
+  private def toString(tuple: (String, String)): String = tuple._1 + " = " + tuple._2;
```
Comment (ddurnev) [inline internal-api/src/main/scala/com/griddynamics/genesis/util/Describer.scala:51]: ; is not necessary here?

File: jclouds/src/main/scala/com/griddynamics/genesis/jclouds/step/JCloudsStep.scala [hunk 4]
```diff
@@ -36,6 +37,14 @@ case class ProvisionVm(roleName: String,
   def isGlobal = true
 
   def roles = Set(roleName)
+
+  override val stepDescription =
+    new Describer("Virtual mashine(s) provisioning")
```
Comment (ddurnev) [inline jclouds/src/main/scala/com/griddynamics/genesis/jclouds/step/JCloudsStep.scala]: Virtual machines

File: jclouds/src/main/scala/com/griddynamics/genesis/jclouds/step/JCloudsStep.scala [hunk 5]
```diff
@@ -43,6 +52,10 @@ case class DestroyVm(roleName: String,
   def isGlobal = true
 
   def roles = Set(roleName)
+
+  override val stepDescription = "Virtual mashine(s) destruction"
```
Comment (ddurnev) [inline jclouds/src/main/scala/com/griddynamics/genesis/jclouds/step/JCloudsStep.scala]: Virtual machines

File: jclouds/src/main/scala/com/griddynamics/genesis/jclouds/step/JCloudsStep.scala (hunk 4 above)
Comment (bugzmanov) [inline jclouds/src/main/scala/com/griddynamics/genesis/jclouds/step/JCloudsStep.scala]: fixed

**Pull request merged**